build/Dependencies.props

```diff
@@ -17,6 +17,7 @@
     <GoogleProtobufPackageVersion>3.5.1</GoogleProtobufPackageVersion>
     <LightGBMPackageVersion>2.2.1.1</LightGBMPackageVersion>
     <MicrosoftMLScoring>1.1.0</MicrosoftMLScoring>
+    <OnnxRuntime>0.1.4-dev-48f58fc4</OnnxRuntime>
```

Comment:
The naming scheme of these properties is as follows:

Take the NuGet package name, and remove all `.` characters. so this should be:

`<MicrosoftMLOnnxRuntime>0.1.4-dev-48f58fc4</MicrosoftMLOnnxRuntime>`

Reply:
Thanks for the catch. Updated.

---
In reply to: [236328447](https://github.com/dotnet/machinelearning/pull/1717#discussion_r236328447) [](ancestors = 236328447)